toolbox/arcgee/data.py

```diff
@@ -372,15 +396,21 @@ def validate_date(date_str, date_format="%Y-%m-%d"):
         raise  # Re-raise the error to stop execution.
 
 
-def download_ee_video(collection, video_args, out_gif, timeout=300, proxies=None):
+def download_ee_video(
+    collection: "ee.ImageCollection",
+    video_args: dict,
+    out_gif: str,
+    timeout: int = 300,
+    proxies: dict = None,
```

Comment:
Same dict comment, plus it needs an `| None`.

It looks like you are not running mypy or pyright to check the annotations.